zeebe-cluster/templates/statefulset.yaml

```diff
@@ -41,15 +41,25 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+          - "--spring.profiles.active=default"
+          - "--spring.config.location=file:./config/zeebe.cfg.yaml"
+          {{- if .Values.extraConfig }}
+          - {{ printf "--spring.config.additional-location=%s" (join "," .Values.extraConfig) | quote }}
+          {{- end }}
         env:
         - name: ZEEBE_LOG_LEVEL
           value: debug
-        - name: ZEEBE_PARTITIONS_COUNT
+        - name: ZEEBE_BROKER_CLUSTER_PARTITIONSCOUNT
```

Comment:
I still don't understand why we have no underscore between PARTITIONSCOUNT, like PARTITIONS_COUNT